cmd/qat_plugin/README.md

```diff
@@ -0,0 +1,102 @@
+# Build and test Intel QuickAssist (QAT) Device Plugin for Kubernetes
+
+### Requirements/Prerequisites
+* Ensure that DPDK drivers are loaded and ready to be used. Please refer to the following for more information on drivers:
+  * [DPDK Getting Started Guide for Linux](https://doc.dpdk.org/guides/linux_gsg/index.html)
+  * [Linux Drivers](http://dpdk.org/doc/guides/linux_gsg/linux_drivers.html)
+* Ensure that the QuickAssist SR-IOV virtual functions are configured. Verify this by running:
+```
+lspci | grep QAT
```

Comment:
All QAT devices don't seem to have their PCI IDs in the ID database. This command is not good to check the availability of, e.g., c62xx.